examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.php

```diff
@@ -0,0 +1,173 @@
+     * ID, dates, and conversion modifier rate specified.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param string $startDateTime the start time of the seasonality adjustment (in
+     *     yyyy-MM-dd HH:mm:ss format) in the account's timezone
+     * @param string $endDateTime the end time of the seasonality adjustment (in
+     *     yyyy-MM-dd HH:mm:ss format) in the account's timezone
+     * @param float $conversionRateModifier the conversion rate adjustment (an increase or a
+     *     decrease), which accounts for estimated changes in conversion rate due to a future event
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $startDateTime,
+        string $endDateTime,
+        float $conversionRateModifier
+    ) {
+        // [START add_bidding_seasonality_adjustment]
+        // Creates a bidding seasonality adjustment.
+        $seasonalityAdjustment = new BiddingSeasonalityAdjustment([
+            // A unique name is required for every seasonality adjustment.
+            'name' => 'Seasonality adjustment #' . Helper::getPrintableDatetime(),
+            // The CHANNEL scope applies the conversionRateModifier to all campaigns of specific
+            // advertising channel types. In this example, the conversionRateModifier will only
+            // apply to Search campaigns. Use the CAMPAIGN scope to instead limit the scope to
+            // specific campaigns.
+            'scope' => SeasonalityEventScope::CHANNEL,
+            'advertising_channel_types' => [AdvertisingChannelType::SEARCH],
+            // If setting scope CAMPAIGN, add individual campaign resource name(s) according to
+            // the commented out line below.
+            // 'campaigns' => ['INSERT_CAMPAIGN_RESOURCE_NAME_HERE'],
+            'start_date_time' => $startDateTime,
+            'end_date_time' => $endDateTime,
+            // The conversionRateModifier is the expected future conversion rate change. When this
+            // field is unset or set to 1.0, no adjustment will be applied to traffic. The allowed
+            // range is 0.1 to 10.0.
+            'conversion_rate_modifier' => $conversionRateModifier
+        ]);
+
+        // Creates a bidding seasonality adjustment operation.
+        $biddingSeasonalityAdjustmentOperation = new BiddingSeasonalityAdjustmentOperation();
+        $biddingSeasonalityAdjustmentOperation->setCreate($seasonalityAdjustment);
+
+        // Submits the bidding seasonality adjustment operation to add the bidding seasonality
+        // adjustment.
+        $biddingSeasonalityAdjustmentServiceClient =
+            $googleAdsClient->getBiddingSeasonalityAdjustmentServiceClient();
+        $response =
+            $biddingSeasonalityAdjustmentServiceClient->mutateBiddingSeasonalityAdjustments(
```

Comment:
This can be in the same line? Or does it exceeds 100 chars?

Reply:
Good catch! It is exactly 100 chars, fixed.